core/store/models/eth_test.go

```diff
@@ -162,7 +162,7 @@ func TestEthTxAttempt_GetSignedTx(t *testing.T) {
 	decodedEncoded := new(bytes.Buffer)
 	require.NoError(t, gotSignedTx.EncodeRLP(decodedEncoded))
 
-	require.Equal(t, signedTx, gotSignedTx)
+	require.Equal(t, signedTx.Hash(), gotSignedTx.Hash())
```

Comment:
The type now includes a `time` attribute to count when it was first seen for anti-spam measurements.